homeassistant/components/conversation/http.py

```diff
@@ -150,6 +151,27 @@ async def websocket_list_agents(
     connection.send_message(websocket_api.result_message(msg["id"], {"agents": agents}))
 
 
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "conversation/sentences/list",
+    }
+)
+@websocket_api.require_admin
+@websocket_api.async_response
+async def websocket_list_sentences(
+    hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict
+) -> None:
+    """List custom registered sentences."""
+    agent = hass.data.get(DATA_DEFAULT_ENTITY)
```

Comment:
If we're sure that the entity is in hass.data we shouldn't use `dict.get(key)` but `dict[key]`.

Reply:
https://github.com/home-assistant/core/pull/133305